src/core/Tools.cpp

```diff
@@ -300,6 +303,54 @@ namespace Tools
         return true;
     }
 
+    QString baseDomain(const QUrl& url)
+    {
+        QString domain = url.host();
```

Comment:
Since this is inappropriately used in icon downloader, we should just move this to Browser code and out of tools to avoid the network dependency in unrelated modules.

Reply:
Marking resolved since this no longer applies.